projects/gui/src/boardview/graphicsboard.cpp

```diff
@@ -108,7 +109,10 @@ void GraphicsBoard::paint(QPainter* painter,
 		rect.moveLeft(rLeft);
 		for (int x = 0; x < m_files; x++)
 		{
-			if ((x % 2) == (y % 2))
+			Chess::Square sq(x,y);
```

Comment:
'int m = m_flipped ? m_files - x - 1 : x;'
'int n = m_flipped ? y : m_ranks - y - 1;'
'Chess::Square sq(m,n);'

'Chess::Square sq(x,y);' should be replaced by the above three line of code

Reply:
Thanks for the heads-up. I just tested with an non-symmetrical setup.

Reply:
I uploaded a corrected version.